wcomponents-core/src/main/java/com/github/bordertech/wcomponents/MutableContainer.java

```diff
@@ -15,6 +15,13 @@
 	 */
 	void add(WComponent component);
 
+	/**
+	 * Adds the given components as "shared" children of this component.
+	 *
+	 * @param components the components to add.
+	 */
+	void addAll(WComponent... components);
```

Comment:
If this can help projects, then I cant see a problem with it. However, to avoid this being a breaking API change (as projects could be implementing their own containers), what if you provided a default implementation on the interface instead.

This could also avoid adding another "package" protected "addAll" method to AbstractWComponent, as we have never liked those methods and they should never of been on AbstractWComponent.

Reply:
I agree in principle with the suggestion. However, removing the method from ```AbstractWComponent``` and adding a default method to the ```MutableContainer``` inteface seems to cause compilation errors. I would have expected that to work.

Reply:
Actually the problem is just removing the method from ```AbstractWComponent```. I added the default method to the interface also, that should stop any issue for any projects that are implementing there own ```MutableContainer```s.

Does this address all of your concerns?

Reply:
I've implemented the requested changes.